app/src/androidTest/java/com/duckduckgo/app/cta/ui/CtaTest.kt

```diff
@@ -209,7 +209,7 @@ class CtaTest {
 
     @Test
     fun whenCanSendPixelAndCtaIsPartOfHistoryThenReturnFalse() {
-        whenever(mockOnboardingStore.onboardingDialogJourney).thenReturn("e:0")
+        whenever(mockOnboardingStore.onboardingDialogJourney).thenReturn("i:0-e:0-s:0")
```

Comment:
Changed so it doesn't matter if the where in the history the cta was (first, last, middle...)